cluster-up/cluster/local/provider.sh

```diff
@@ -32,8 +32,7 @@ function up() {
 }
 
 function prepare_config() {
-    PROVIDER_CONFIG_FILE_PATH="${BASE_PATH}/$KUBEVIRT_PROVIDER/config-provider-$KUBEVIRT_PROVIDER.sh"
-    cat > "$PROVIDER_CONFIG_FILE_PATH" <<EOF
+    cat >hack/config-provider-local.sh <<EOF
```

Comment:
This reverts a needed fix, see please 
https://github.com/kubevirt/kubevirt/issues/4041
i suggest please to not include this file in this commit (`make generate` did this change for you)

Reply:
Yes `make generate` made this change for me, removed now